<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QWidget" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>11</red>
        <green>118</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>11</red>
        <green>118</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>11</red>
        <green>118</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans CJK SC Regular</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>末端试水系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ProjectManager{
background-image: url(:/pic/res/login_global_background.png);
}</string>
  </property>
  <widget class="QLabel" name="projectManageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC Regular</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>项目管理</string>
   </property>
  </widget>
  <widget class="QFrame" name="projectManagerBackgroundFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>961</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#projectManagerBackgroundFrame{
background-image: url(:/pic/res/rectangle_background.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="newProjectPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#newProjectPushButton{
color: rgb(255, 255, 255);
background-color: rgb(11, 118, 100);
border-radius:5px
}</string>
    </property>
    <property name="text">
     <string>新建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openProjectPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#openProjectPushButton{
color: rgb(255, 255, 255);
background-color: rgb(11, 118, 100);
border-radius:5px
}</string>
    </property>
    <property name="text">
     <string>打开项目表</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectNameLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>35</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>项目名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectLocationLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>35</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>项目位置</string>
    </property>
   </widget>
   <widget class="QLabel" name="noteLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>备     注</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="projectNoteTextEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>660</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="projectNameLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>260</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="projectLocationLineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>281</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="userManagerBackgroundFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>961</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#userManagerBackgroundFrame{
background-image: url(:/pic/res/rectangle_background.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="newUserPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#newUserPushButton {
color: rgb(255, 255, 255);
background-color: rgb(11, 118, 100);
border-radius:5px
}</string>
    </property>
    <property name="text">
     <string>新建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openUserPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#openUserPushButton{
color: rgb(255, 255, 255);
background-color: rgb(11, 118, 100);
border-radius:5px
}</string>
    </property>
    <property name="text">
     <string>打开用户表</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="userProjectBelongZToLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>所属项目</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>85</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密    码</string>
    </property>
   </widget>
   <widget class="QLabel" name="userTypeLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>85</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="userProjectBelongToCombox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>25</y>
      <width>171</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="userTypeCombox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>171</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>25</y>
      <width>260</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="userPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>260</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="deviceManagerBackgroundFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>961</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#deviceManagerBackgroundFrame{
background-image: url(:/pic/res/rectangle_background.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="openDevicePushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#openDevicePushButton{
color: rgb(255, 255, 255);
background-color: rgb(11, 118, 100);
border-radius:5px
}</string>
    </property>
    <property name="text">
     <string>打开设备表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newDevicePushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#newDevicePushButton {
color: rgb(255, 255, 255);
background-color: rgb(11, 118, 100);
border-radius:5px
}</string>
    </property>
    <property name="text">
     <string>新建</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceIdLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>25</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备  ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceAdressLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>135</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceProjectBelongToLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>25</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>所属项目</string>
    </property>
   </widget>
   <widget class="QComboBox" name="deviceProjectBelongToComboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>181</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceAdressLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>125</y>
      <width>260</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceIdLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>260</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="kValueLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>K         值</string>
    </property>
   </widget>
   <widget class="QComboBox" name="kValueComboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>75</y>
      <width>181</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="deviceNameLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>75</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceNameLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>75</y>
      <width>261</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="userManageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC Regular</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceManageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC Regular</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设备管理</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
